sormas-api/src/main/resources/strings.properties

```diff
@@ -521,15 +524,15 @@ infoUploadDocumentTemplate=Select a ".docx"-Document Template you would like to
-infoHowToMergeCases = You can choose between two options when reviewing potentially duplicate cases:
-infoCalculateCompleteness = The <i>Calculate Completeness</i> button on top can be used to calculate the completeness values for all cases in the table. This is only necessary if there are cases in the list that do not yet have a completeness value. Normally, this value is automatically updated whenever the details of a case or one of its contacts or samples change.
-infoCaseCompleteness = Completeness is a measurement of how much vital data has already been entered into the case. This is composed of the following, weighted by their importance:<br/><ul><li>Has the case been classified?</li><li>Has the case been investigated?</li><li>Has at least one sample been taken?</li><li>Has at least one symptom been documented?</li><li>Has at least one contact been created?</li><li>Is there an outcome for the case?</li><li>Does the case have a birth date or age?</li><li>Does the case have a sex?</li><li>Has an onset date been specified?</li></ul>
-infoCompletenessMerge = You can use this measurement to get an idea about which case you should use as the one that remains active when merging. However, please make sure that you still have a detailed look at both cases to avoid mistakes.
-infoMergeIgnoreRegion = Selecting this option excludes the region as a criteria for detecting duplicates. Merging and picking are deactivated here because the probability of a real duplicate beyond region boundaries is very low. If a real duplicate is detected, this case can be deleted manually via the case directory.
+infoMergingExplanation = This view is designed to assist you in detecting and merging duplicate entities. entity are always displayed as pairs, the entity with the earlier creation date being on top.<br/><br/>You can use the creation date filter to only view potential duplicates that were created in a specific time period. The creation date is automatically generated when a entity is entered into SORMAS and can not be changed by users. Therefore, once you have reviewed all duplicates from a specific time period in the past, you will never have to do it again as there will be no new entities created during this time frame.<br/><br/>Please note that most likely not all displayed entities are actual duplicates. There is the potential for false positives, so <b>please thoroughly review the entities before merging them!</b> In addition to the information provided in the table, you can also click on the IDs of the entities to open them in a new tab. Keep in mind that it is much worse to falsely merge unique entities together than to have some duplicates in the system.
```

Comment:
Error in this string:

[...] "entities. entity are always displayed as pairs, [...]" => "entities. Entities are always displayed as pairs,"